omod/src/test/java/org/openmrs/module/fhir2/providers/LocationFhirResourceProviderTest.java

```diff
@@ -222,4 +231,17 @@ public void findLocationsByWrongPostalCode_shouldReturnBundleWithEmptyEntries()
 		assertThat(results.getEntry(), is(empty()));
 	}
 	
+	@Test
+	public void findLocationsByTags_shouldReturnMatchingBundleOfLocations() {
+		TokenParam tag = new TokenParam(FhirConstants.OPENMRS_FHIR_EXT_LOCATION_TAG, LOGIN_LOCATION_TAG_NAME);
+		when(locationService.findLocationsByTag(tag)).thenReturn(Collections.singletonList(location));
+		StringParam param = new StringParam();
+		param.setValue(LOCATION_NAME);
```

Comment:
You should remove this code block. It has no use or do you have any use for this?